FE/src/components/MedicineItem/MedicineItem.jsx

```diff
@@ -1,41 +1,82 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import { Button, Card, CardContent, CardHeader, Grid, IconButton, Typography } from '@mui/material'
-import VaccinesIcon from '@mui/icons-material/Vaccines'
-import DeleteSweepIcon from '@mui/icons-material/DeleteSweep';
+import { React, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import {
+  Card,
+  CardContent,
+  Chip,
+  Grid,
+  IconButton,
+  Typography,
+} from "@mui/material";
+import DeleteSweepIcon from "@mui/icons-material/DeleteSweep";
 
-const MedicineItem = props => {
-    return (
-        <Card sx={{ width: '70rem' }}>
-            <CardHeader
-                avatar={
-                    <VaccinesIcon />
-                }
-                title={props.name}
-                titleTypographyProps={{ variant: 'h6' }}
-                action={
-                    <IconButton>
-                        <DeleteSweepIcon sx={{ color: 'red' }} />
-                    </IconButton>
-                }
+const MedicineItem = ({
+  medicineName = "Panadol",
+  hospitalName = "VinMec",
+  expirationDate = "10/12/2023",
+  status = "LISTED",
+  handleDelete = () => {},
+}) => {
+  const [badgeColor, setBadgeColor] = useState(() => {
```

Comment:
What about a dict that contains status as a key and label as a value? I think it can help